spec/services/translation_service/api/translation_spec.rb

```diff
@@ -0,0 +1,191 @@
+require_relative '../api'
+
+describe TranslationService::API::Translations do
```

Comment:
This is in line with our other services. 
This way, it is easier to understand what test is in question since our folder structure is separated by function (and different folders might contain files with similar names)
some_service
> api
+->file1_spec.rb
> datatypes
+->file1_spec.rb
> store
+->file1_spec.rb